src/main/java/org/profit/telegrambot/model/Category.java

```diff
@@ -0,0 +1,16 @@
+package org.profit.telegrambot.model;
+
+import lombok.*;
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@NoArgsConstructor
+public class Category {
```

Comment:
Check out this one: https://metanit.com/java/tutorial/3.18.php

Despite plain-old lombok seems better (IMHO), sometimes it is not possible to use it. So, the `record`s worth being familiar with.